.github/workflows/deploy-preview.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           flyctl secrets set GRPC_AUTH_TOKEN="$GRPC_AUTH_TOKEN" --app "$APP_NAME" --stage
           flyctl secrets set HOTORNOT_GOOGLE_CLIENT_SECRET="$HOTORNOT_GOOGLE_CLIENT_SECRET" --app "$APP_NAME" --stage
           flyctl secrets set ICPUMPFUN_GOOGLE_CLIENT_SECRET="$ICPUMPFUN_GOOGLE_CLIENT_SECRET" --app "$APP_NAME" --stage
-          flyctl deploy --app $APP_NAME
+          flyctl deploy --app $APP_NAME --depot=false
```

Comment:
This should not be necessary? cc: @ravibazz

Reply:
yep, was temp, will undo